OsmAnd/src/net/osmand/plus/settings/backend/OsmandSettings.java

```diff
@@ -3150,4 +3150,21 @@ public void setQuickActions(HashMap<String, Boolean> quickActions, ApplicationMo
 			setPreference(QUICK_ACTION.getId(), actionState, mode);
 		}
 	}
+
+	public final CommonPreference<String> SPEED_SENSOR_WRITE_TO_TRACK_DEVICE =
+			new StringPreference(this, "speed_sensor_write_to_track_device", "").makeProfile().cache();
+
+	public final CommonPreference<String> CADENCE_SENSOR_WRITE_TO_TRACK_DEVICE =
+			new StringPreference(this, "cadence_sensor_write_to_track_device", "").makeProfile().cache();
+
+	public final CommonPreference<String> POWER_SENSOR_WRITE_TO_TRACK_DEVICE =
+			new StringPreference(this, "power_sensor_write_to_track_device", "").makeProfile().cache();
+
+	public final CommonPreference<String> HEART_RATE_SENSOR_WRITE_TO_TRACK_DEVICE =
```

Comment:
Is data writing from two devices supported?

Reply:
No. The task was one type - one device